src/mgmt/srv.c

```diff
@@ -322,9 +322,8 @@ ds_mgmt_hdlr_svc_rip(crt_rpc_t *rpc)
 	 * the yield below is to workaround an ofi err msg at client-side -
 	 * fi_cq_readerr got err: 5(Input/output error) ..
 	 */
-	int i;
 
-	for (i = 0; i < 200; i++) {
+	for (int i = 0; i < 200; i++) {
```

Comment:
should the declaration of i be move to the beginning rather than in the loop?
to be consistent with other changes in this PR like:
https://github.com/daos-stack/daos/pull/13094/files#diff-f733e5ebd07cfd256e6f6cf2f5e934e5b88ed055b1520d558a71ddb0be10d746R600